generators/app/templates/src/app/components/FormInput/styles.module.scss

```diff
@@ -0,0 +1,28 @@
+@import 'variables/_colors';
+
+.input {
+  background-color: $white;
+  border-radius: 5px;
+  border-bottom: 2px solid $gray;
+  height: 30px;
+  padding: 0 12px;
+
+  &:focus {
+    border-bottom: 2px solid $blue;
+  }
+
+  &.error {
+    border-bottom: 2px solid $red;
+  }
+}
+
+.error-text {
+  color: $red;
+  font-size: 12px;
+  height: 20px;
+  opacity: 0;
+
+  &.visible {
+    opacity: 1;
+  }
```

Comment:
What about changing this and use `display` or `visibility` attributes instead? Take into account that `display: none;` removes the elem from the layout while `visibility: hidden;` hides it but reserves the space. I think `opacity` would make sense when dealing with a transition from transparent to opaque.